style.css

```diff
@@ -0,0 +1,119 @@
+* {
+    box-sizing: border-box;
+    font-family: 'IBM Plex Sans', sans-serif;
+}
+/* 
+html,
+body{
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+} */
+
+body {
+    margin: 0;
+    background: rgb(116,255,0);
+    background: linear-gradient(0deg, rgba(116,255,0,1) 0%, rgba(255,255,255,1) 100%) fixed;
+}
+
+h1 {
+    font-size: 25px;
+    text-transform: uppercase;
+    padding-top: 3px;
+    letter-spacing: 1px;
+}
+
+h2 {
+    font-size: 20px;
+    padding-bottom: 6px;
+    text-transform: uppercase;
+    letter-spacing: 3px;
+}
+
+h3 {
+    font-size: 16px;
+    margin-bottom: 0;
+}
+
+h4 {
+    color: #333333;
+    font-style: italic;
+    font-weight: 400;
+    font-size: 15px;
+    margin-bottom: 10px;
+    margin-top: 5px;
+}
+
+p,
+li {
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 1.7;
+}
+
+p {
+    padding-left: 30px; 
+    margin-top: 7px;
+}
+
+ul {
+    list-style: none;
+    padding-left: 35px;
+    margin-top: 7px;
+
+}
+
+a {
+    color: black;
+    text-decoration: none;
+}
+
+.wrapper {
+    /* height: 100%; */
+    max-width: 1200px;
+    /* max-height: 800px; */
+    /* border: 1px solid red; */
+    margin: auto;
+    position: relative;
+}
+
+/* info */
+#info {
+    /* background-color: #6480E3; */
+    border-right: 1px solid black;
+    width: 200px;
+    height: 100%;
+    padding-left: 25px;
+    position: fixed;
+}
+
+#info ul{
+    padding-left: 0;
+}
+
+/* content */
+#cotent-wrap {
+    width: 100%;
+    /* height: 100%; */
+    /* max-height: 800px; */
+    padding-left: 200px;
+    /* overflow: auto; */
+    /* position: relative; */
+}
+
+.column {
+    width: 50%;
+    float: left;
+}
+
+#eas{
+    border-left: 1px solid black;
+}
+
+.inner-col {
+    padding: 6px 35px 40px 20px;
+    border-bottom: 1px solid black;
+}
+.inner-col:last-child{
```

Comment:
While I think it's great you're already familiar with pseudoclasses 🙌🏻, I like to [avoid writing CSS that immediately has to be undone](https://mica-web.github.io/learn/css/best-practices#dont-write-css-in-such-a-way-that-youre-immediately-overwriting-it), which is what's happening here -- your line 118 is just undoing the styles you wrote on line 115.

If you made this a top border and used the sibling combinator, you would avoid this:

```
.inner-col {
    padding: 6px 35px 40px 20px;
}
.inner-col + .inner-col {
    border-top: 1px solid black;
}
```

So, in your CSS you're adding a bottom border to all your `inner-col` selectors and then removing it on the last one. In my version, I'm _only_ adding a top border to `inner-col` selectors that follow another `inner-col` selector. Learn more about the [sibling combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator) on MDN. 💡

Reply:
Also, to be clear -- your CSS isn't wrong. This is merely a refinement of that idea.